packages/terra-table/package.json

```diff
@@ -23,14 +23,17 @@
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
     "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+    "react-dom": "^16.8.5",
+    "react-intl": "2 || 3 || 4 || 5"
```

Comment:
Was this change necessary?  Do we actually support all of these versions?

Reply:
Paging @sdadn for this comment.

Reply:
We support react-intl 2 through 5. The original syntax was `>= 2, < 6` but this makes it more clearer.